apps/src/lib/levelbuilder/rubrics/EvidenceDescriptions.jsx

```diff
@@ -21,36 +22,52 @@ export default function EvidenceDescriptions({
       </div>
       <EvidenceDescriptionsRow
         isAiEnabled={learningGoalData.aiEnabled}
-        evidenceLabel={UNDERSTANDING_LEVEL_STRINGS[3]}
+        evidenceLabel={
+          UNDERSTANDING_LEVEL_STRINGS[RubricUnderstandingLevels.EXTENSIVE]
+        }
         evidenceLevelData={
-          learningGoalData.learningGoalEvidenceLevelsAttributes[3]
+          learningGoalData.learningGoalEvidenceLevelsAttributes[
+            RubricUnderstandingLevels.EXTENSIVE
+          ]
```

Comment:
This is based on feedback from Bethany to get rid of the magic numbers.

Reply:
thank you!